common/test/acceptance/pages/lms/programs.py

```diff
@@ -24,8 +26,8 @@ def is_sidebar_present(self):
 
 class ProgramDetailsPage(PageObject):
     """Program details page."""
-    program_id = 123
-    url = BASE_URL + '/dashboard/programs/{}/program-name/'.format(program_id)
+    program_uuid = str(uuid4())
+    url = BASE_URL + '/dashboard/programs/{}/'.format(program_uuid)
```

Comment:
Just for readability, can this be all done with a format '....'.format(BASE_URL, program_uuid).  Just so we don't have strong concatenation mixed in with a format.